recipes/IEMOCAP/emotion_recognition/hparams/train_with_wav2vec2.yaml

```diff
@@ -0,0 +1,116 @@
+# ########################################
+# Emotion recognition from speech using wav2vec2
+#
+#  * Authors: Yingzhi WANG
+# ########################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 1968
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+experiment: emotion_recognition
+# Set up folders for reading from and writing to
+# Dataset will be downloaded to the `data_folder`
+data_folder: !PLACEHOLDER # e.g., /storage/IEMOCAP/IEMOCAP_ahsn_leave-two-speaker-out
+output_folder: !ref results/wav2vec/<seed>
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# URL for the wav2vec2 model, you can change to benchmark diffrenet models
+wav2vec2_hub: "facebook/wav2vec2-base-960h"
```

Comment:
why are you using the non fine tuned one on ASR ?
did you see the difference

Reply:
yes you are right I am carrying out more experiments to be precise